frontend/libs/studio-components/src/components/StudioTextField/StudioTextfieldSchema/StudioTextfieldSchema.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo, useState } from 'react';
+import {
+  StudioToggableTextfield,
+  type StudioToggableTextfieldProps,
+} from '../StudioToggableTextfield';
+import type { JsonSchema } from '../../../../../../packages/shared/src/types/JsonSchema';
+import { useTranslation } from 'react-i18next';
+import { StudioJSONValidatorUtils } from '../StudioJSONValidatorUtils';
+
+export type StudioTextfieldSchemaProps = {
+  schema: JsonSchema;
+  propertyPath: string;
+  jsonValidator: any;
```

Comment:
I can checkout the pull request and see if I can make this more generic, so we can avoid using an explicit "any" type. 😊